src/pint/utils.py

```diff
@@ -1667,3 +1667,17 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
+    import pint
+    import getpass
+    import platform
+    import datetime
+
+    s = f"Created: {datetime.datetime.now().isoformat()}\nPINT_version: {pint.__version__}\nUser: {getpass.getuser()}\nHost: {platform.node()}\nOS: {platform.platform()}"
```

Comment:
A triple-quoted string would be much easier here. `textwrap.dedent` exists to clean up spurious indentation in such strings.